src/Feature/Keyboard/Feature_Keyboard.re

```diff
@@ -0,0 +1,77 @@
+open Oni_Core;
+
+// Placeholder model type
+type model = int;
+let initial: model = 0;
+
+[@deriving show]
+type command =
+  | ShowEmojiAndSymbols;
+
+[@deriving show]
+type msg =
+  | Empty(unit)
+  | Command(command);
+
+type outmsg =
+  | Nothing
+  | Effect(Isolinear.Effect.t(msg));
+
+module Commands = {
+  open Feature_Commands.Schema;
+
+  let showEmojiAndSymbols =
+    define(
+      ~category="Keyboard",
+      ~title="Input Emoji & Symbols",
+      ~isEnabledWhen="isMac" |> WhenExpr.parse,
+      "oni2.keyboard.emojiAndSymbols", // This doesn't exist in VSCode
+      Command(ShowEmojiAndSymbols),
+    );
+};
+
+module MenuItems = {
+  open MenuBar.Schema;
+  module Edit = {
+    let emojiAndSymbols =
+      command(~title="Emoji & Symbols", Commands.showEmojiAndSymbols);
+  };
+};
+
+module Keybindings = {
+  open Feature_Input.Schema;
+
+  let emojiAndSymbols =
+    bind(
+      ~key="<C-D-Space>",
```

Comment:
@bryphe here's the keybind definition that doesn't seem to be working. I'll try to debug it

Reply:
Oh, interesting - did your change here address this? https://github.com/revery-ui/revery/pull/1079

Reply:
Just tried the latest and saw that it is working - nice!